app/src/main/java/com/android/streetworkapp/ui/authentication/GoogleAuthButton.kt

```diff
@@ -45,7 +45,10 @@ fun GoogleAuthButton(
       modifier =
           Modifier.width(250.dp)
               .height(40.dp)
-              .border(width = 1.dp, color = LightGray, shape = RoundedCornerShape(20.dp))
+              .border(
+                  width = 5.dp,
+                  color = ColorPalette.INTERACTION_COLOR_DARK,
```

Comment:
Suggestion: I'm not sure about the design of this button; the blue border color isn't very common. A button with a thin gray border would be more consistent with what users are accustomed to for a "Sign in with Google" option. 

You could aim to change the button to look like this one
<img width="218" alt="image" src="https://github.com/user-attachments/assets/8093dc58-5fd7-4311-b8f9-6acb62b1b492">

You should also update the tutorial screens with the new button look.

Reply:
new button look ??? which button ? the blue one in the tuto screen ?

Reply:
Changed the border to be black